admin-dev/themes/new-theme/scss/pages/_modules.scss

```diff
@@ -569,22 +566,99 @@ form#importDropzone {
     display: table;
     width: 100%;
     padding: 5px 0;
-    margin-bottom: -1px;
-    border: solid 1px #9ed0ec;
+    border-bottom: solid 1px #bbcdd2;
```

Comment:
use variable for color please. $gray-light

Reply:
:ok_hand: